components/ClearBoard.tsx

```diff
@@ -0,0 +1,28 @@
+import { Actions, useStore } from "./Store";
+import { getLocalStorage, clearLocalStorage } from "../utils";
+
+export default function ClearBoard({ id }) {
+  const { dispatch } = useStore();
+
+  const onClickClearBoard = () => {
+    const boardStorage = getLocalStorage(id);
+    const cleanBoard = { ...boardStorage, answers: [] };
+
+    clearLocalStorage(id);
+
+    dispatch({
+      type: Actions.HydrateBoard,
+      payload: { board: cleanBoard },
+    });
+  };
+
+  return (
+    <img
+      className="clear-board-icon"
```

Comment:
Also, for accesibility (so the button is available when clicking tab) You'll need to give it a tabIndex and role (since it's not a button)
See example in Square.tsx:
```
const Container = styled(motion.div).attrs({ role: "button", tabIndex: 0 })
```
and also make the callback trigger on keyPress as well as onClick (so it's keyboard accessible)

Also from Square.tsx:
```
<Container
       ...
      onClick={onSelect}
      onKeyPress={onSelect}
    >
```